framework/public/public_test.go

```diff
@@ -144,8 +149,20 @@ func TestGetChangeGet(t *testing.T) {
 	is.NoErr(err)
 	is.Equal(200, res.Status())
 	is.Equal(res.Body().Bytes(), favicon2)
-	// is.Equal(result.Stdout(), "")
-	// is.Equal(result.Stderr(), "")
+
+	res, err = app.Get("/default.css")
+	is.NoErr(err)
+	is.Equal(200, res.Status())
+	is.Equal(res.Body().Bytes(), defaultCss)
+
+	// default.css changed
+	defaultCss2 := []byte("/* changed default.css */")
+	td.BFiles["public/default.css"] = defaultCss2
+	is.NoErr(td.Write(ctx))
+	res, err = app.Get("/default.css")
+	is.NoErr(err)
+	is.Equal(200, res.Status())
+	is.Equal(res.Body().Bytes(), defaultCss2)
```

Comment:
It's a bit hard to follow these changes because you've added new conditions to existing tests. 

Let's try to only touch existing tests that need to be updated for the new behavior, then add new tests for the new behavior.

Reply:
Reverted. No need to update on `public.go` just update on `ssr.go`